ivy/functional/frontends/paddle/tensor/linalg.py

```diff
@@ -122,6 +122,16 @@ def solve(x1, x2, name=None):
     return ivy.solve(x1, x2)
 
 
+# cholesky_solve
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def cholesky_solve(y, x, /, *, upper=False, name=None):
```

Comment:
The original function puts `x` before `y`. We should do so as well; the signature should be exactly the same.

Reply:
Actually that was a bug in their documentation. I found that, reported and fixed that to them as well. thats the reason for that confusion. link to the paddle paddle bug : https://github.com/PaddlePaddle/Paddle/issues/55998